tests/Load/aws-execute-load-tests.sh

```
@@ -0,0 +1,154 @@
+#!/bin/bash
+set -e
+
+if [ -f "./tests/Load/config.sh" ]; then
+  . ./tests/Load/config.sh
```

Shellcheck

[info] Not following: ./tests/Load/config.sh was not specified as input (see shellcheck -x).

(SC1091)